### DISCUSSION TITLE
Addition Of  a Lore Map

### DISCUSSION THREAD

**Discussion by Z1kou** [sunflower-land/sunflower-land#1211] [Ideas]
*We at Pixelized Crib are exploring new features to add to the game, and today I am gonna open several discussion's for every single one of them so sit down get a drink and buckle up because i need your spicy opinion's everyone !*

 # Feature Details

Adding a lore map much like goblin village but on the left side of the map, This map would be purely for lore and scenary with a new take on the SFL world that is in night mode and other aesthetic components and interactible piece's of lore ( potentially adding Dev's NPC to it ), and potentially having the dev's NPC tell about a future community event (Example Addition of Axolotl's)

# Purpose

The purpose of the map is to add a sort of resting place for farmer's from the crazy goblin x farmer's war and a sort of get away from all the responsibilities we have as a farmer !

This map would  include:
-Night mode 
-Campfire and camp logs
-Fire flies and other night creatures
-Grafities on mountain walls pillars and rocks that are interactible and contains lore
-Potential teaser of an upcoming community event 

# Potential Direction:

We could either have it for the aesthetic side of it and as new content that would not need much interaction from the dev since its purely front end with no ingame function other than being a lore area that is eye appealing

Potentially adding it as a place to get hints on future upcoming updates 

Potentially adding an entry fee to this area that would be just an incentive to burning some ressource's

What are your thoughts Farmers :farmerEmoji:?

**Comment by JcEii** [comment]
I like the idea. I think those included on the map have different biomes to create uniqueness (like night = swamp biome, mountain = snowy mountain). It makes the game more exciting for those players who like lore, and hopefully, this will be included in the features of the game in the future. up, up!

**Reply by JcEii** [reply]
- Seasons
- Biomes
- Country Based (e.g. Medieval period - Countryside period)

**Comment by Aminbtdds** [comment]
Great idea👌
We need a place in-game to simply show to our players, what is gonna happen in future, ofc it will be more intresting if our new place have aesthetic side, it adds value to overall beauty of sunflowerland.
P.s.
They are lots of players which can not use discord or engage properly with servers, this will solve the issue of not being up to date about progresses and further plans.
